README.md

```
@@ -1,23 +1,38 @@
+
+Auto-sync fork is a GitHub Actions workflow that automatically syncs your fork with the upstream repository.
+
+## How to install?
+
+### Auto-run using GitHub Actions (Recommended)
 
 - Go to repository settings
 - Select Secrets and variables → Actions
 - Add new secrets with name
-  - PERSONAL_TOKEN: your github account token, not repository token. example: ghp_xxx
+  - **PERSONAL_TOKEN**: your github account token/github organization token, not repository token. example: 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)